lib/inspector/flutter_widget.dart

```diff
@@ -0,0 +1,124 @@
+library flutter_widget;
+
+import 'dart:convert';
+
+import 'package:devtools/ui/icons.dart';
+import 'package:devtools/utils.dart';
+import 'package:http/http.dart';
+
+class Category {
+  const Category(this.label, this.icon);
+
+  static const Category ACCESSIBILITY =
```

Comment:
Is it dart style to have all-caps class members? When I've written the java style all-caps const's, I've received feedback to avoid them. However, IIRC that was for flutter code I wrote, so this could be a different situation?